.env

```diff
@@ -67,3 +67,6 @@ NEXT_PUBLIC_SITE_IS_PREVIEW=false
 JAEGER_UI_PORT=16686
 JAEGER_OLTP_PORT=4318
 TRACING_ENABLED=1
+
+# BREVO
+BREVO_API_KEY="PLEASE OVERWRITE IN .env.local FILE"
```

Comment:
Couldn't you remove the variable?  Then the API Config validation should throw a validation error that the key needs to be set.

And I would say we describe in the README what needs to be done to start the project (setting the BREVO_API_KEY in the .env.local file)